### DISCUSSION TITLE
I don 't know why follow jstree plugin doesn 't work, it should create buttons for every nodes and assign an action

### DISCUSSION THREAD

**Discussion by user-redans** [vakata/jstree#2642] [General]
I've tried to execute this [fiddle](https://jsfiddle.net/blablabla/2tzesu1x/3/) and it s running without any errors, but when I have clicked on any nodes buttons, "wholerow" plugin doesn't work. I've found this plugin [here](http://www.sldesign.url.tw/JsTree/index.html).... seems that doesn't work neither in original post and I don 't know why? I want to triggered defined action when one of those buttons are pressed.

I've write this post on stackoverflow , [this ](https://stackoverflow.com/questions/73294007/i-don-t-know-why-follow-jstree-plugin-doesn-t-work)is my topic from there

Basically, I need for every leaf of the tree to create a button, like in my jsfiddle example and assign it an action event.....
